tools/device_management.py

```diff
@@ -51,6 +51,9 @@ def __init__(self, *args, **kwargs):
             "Create or transform a manifest. "
             "Use {} [command] -h for help on each command."
         ).format(self.parser.prog)
+        subparserActionLength = len(self.parser._subparsers._actions)
+        initParser = self.parser._subparsers._actions[subparserActionLength-1].choices['init']
+        initParser.add_argument('--disable-cert', help='Disable writing the developer certificate to mbed_cloud_dev_credentials.c')
```

Comment:
This is going to sound like bikeshedding, but hear me out I have a good reason 😄 
The `mbed dm init` command creates two certificates in the user's project:
- The developer certificate
- The update certificate

Could we change this argument to explicitly mention we are disabling the download of the developer certificate?
```suggestion
        initParser.add_argument('--no-developer-cert', help='Do not download the developer certificate')
```

I went with the `--no-*` style syntax in my above suggestion because it mirrors the `--no-requirements` option that was recently added to Mbed CLI: https://github.com/ARMmbed/mbed-cli/blob/master/mbed/mbed.py#L2210

I also dropped the mention of the exact file just in case that file name ever changes. Just makes the help text a bit more general.

Reply:
To be honest, I was expecting comments and suggestions regarding the content of the arguments. I wasnt too happy with it myself :) 
Good suggestions, I agree..